test/unit/tss2_policy.c

```diff
@@ -0,0 +1,287 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*******************************************************************************
+ * Copyright 2017-2018, Fraunhofer SIT sponsored by Infineon Technologies AG
+ * All rights reserved.
+ *******************************************************************************/
+
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include <limits.h>
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <setjmp.h>
+#include <cmocka.h>
+
+#if defined(__linux__)
+#include <linux/limits.h>
+#endif
+
+#include "tss2_policy.h"
+
+#define LOGMODULE test
+#include "util/log.h"
+
+#include "test/data/test-fapi-policies.h"
+#include "util/aux_util.h"
+
+#define TEST_LAYER        TSS2_RC_LAYER(0)
```

Comment:
Because of `#define TSS2_TPM_RC_LAYER    TSS2_RC_LAYER(0)` there would be equal error codes.
We should use a value not already defined in tss2_common.h.

Reply:
Whoops I meant to do that.